static_report/src/components/ComparisonReport/CRAssertions.tsx

```diff
@@ -0,0 +1,275 @@
+import { Flex, Text, Center, Icon } from '@chakra-ui/react';
+import {
+  FiCheck,
+  FiX,
+  FiArrowUpCircle,
+  FiArrowDownCircle,
+} from 'react-icons/fi';
+import isString from 'lodash/isString';
+import partial from 'lodash/partial';
+
+import { getComparisonAssertions } from '../../utils/assertion';
+import type { ComparisonReportSchema } from '../../types';
+
+const getAssertionValue = partial((value: string | number) =>
+  isString(value) ? 0 : value,
+);
+
+export function CRAssertions({
```

Comment:
Are we able to add a `<UI_NAME>` suffix to this component name? Else it's hard to know by name what this does in the UI level. 
Is this a matrix (e.g. tabular/table data,  a list? rows)

Reply:
Agree with you, lemme change the file name.